packages/polaris-core/src/middlewares/snapshot-middleware.ts

```diff
@@ -44,7 +51,7 @@ export class SnapshotMiddleware {
         context: PolarisGraphQLContext,
         result: any,
     ) {
-        if (context.snapshotContext == null || context.snapshotContext.startIndex === 0) {
+        if (context.snapshotContext == null || !context.snapshotContext.startIndex) {
```

Comment:
i think that for sake of readability undo that and leave it === 0, unless its for checking for nulls/undefind etc